### PR TITLE
prune flake8 ignore list

### PR DESCRIPTION
**Pull request by ploxiln**
re-enable some flake8 rules which cause the least code churn

### CODE REVIEW

**Comment by ploxiln** [comment]
DeepSource is finding stuff like improper indent, on the lines I'm fixing other style issues on. It's better to not re-indent a significant part of the file at the same time though.

**Comment by ploxiln** [comment]
The other thing DeepSource is flagging is "redefining name from outer scope" for a bunch of render function kwargs in `webapp/graphite/render/functions.py`, but those will never be fixed because it's the (public api) render function "color()" or "alpha()" that is shadowed by the argument "color" or "alpha" for another (public api) render function.

File: check-dependencies.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
+# flake8: noqa
```
Comment (piotr1212) [inline check-dependencies.py]: Could you narrow this one down with the specific rule to ignore? You can do with `# noqa: E123

File: webapp/graphite/account/ldapBackend.py [hunk 2]
```diff
@@ -12,7 +12,9 @@
 See the License for the specific language governing permissions and
 limitations under the License."""
 
-import ldap, traceback
+import traceback
+
```
Comment (cbowman0) [inline webapp/graphite/account/ldapBackend.py]: Just curious why the blank line here?

**Comment by DanCech** [comment]
Thanks for tackling this!

File: check-dependencies.py (hunk 1 above)
Comment (ploxiln) [reply check-dependencies.py]: In this file it's just a bunch of "F401 imported but unused" (which is the intention of this script). You can't ignore a specific rule for an entire file, so I'll have to add  `# noqa: F401` to each line. But I just remembered there's another trick using `__all__` which may or may not be cleaner :)

File: webapp/graphite/account/ldapBackend.py (hunk 2 above)
Comment (ploxiln) [reply webapp/graphite/account/ldapBackend.py]: It is a somewhat less consistent convention, to put standard library imports first, then a blank line, then third-party imports, then a blank line, then project-internal imports.

**Comment by ploxiln** [comment]
I can easily squash down these commits if desired, just let me know.

File: webapp/graphite/account/ldapBackend.py (hunk 2 above)
Comment (cbowman0) [reply webapp/graphite/account/ldapBackend.py]: Ok.  Thanks for the explanation.

**Comment by ploxiln** [comment]
(I squashed down to 3 commits, which seemed appropriate.)

**Comment by deniszh** [comment]
Thanks a lot again, @ploxiln !
I will merge it today, I want to test backport bot on it.

**Comment by ploxiln** [comment]
Thank you for maintaining graphite :)

**Pull request merged**